services/workflows-service/src/common/filters/AllExceptions.filter.ts

```diff
@@ -23,6 +23,7 @@ export class AllExceptionsFilter extends BaseExceptionFilter {
 
   private _handleHttpErrorResponse(exception: unknown, request: Request, response: Response) {
     const serverError = this.getHttpException(exception);
+    const errors = exception instanceof AjvValidationError ? exception.serializeErrors() : [];
```

Comment:
why dont we use ajv.errorsText?
i've seen also seen this usage : services/workflows-service/src/workflow/workflow.service.ts:861
<img width="699" alt="image" src="https://github.com/ballerine-io/ballerine/assets/6435752/068d1cb8-e037-438e-8332-5416656b25d9">

Reply:
So it would be better to use our custom error with a factory method for each provider...